boards/seeed/xiao_esp32s3/xiao_esp32s3_procpu_sense.dts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2023 Seeed Studio inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include "xiao_esp32s3_procpu_common.dtsi"
+
+/ {
+	model = "Seeed Xiao ESP32S3 PROCPU Sense";
+	compatible = "seeed,xiao-esp32s3";
+
+	chosen {
+		zephyr,camera = &lcd_cam;
+	};
+};
+
+&i2c1 {
+	status = "okay";
+	clock-frequency = <I2C_BITRATE_STANDARD>;
+	pinctrl-0 = <&i2c1_default>;
+	pinctrl-names = "default";
+
+	ov2640: ov2640@30 {
+		compatible = "ovti,ov2640";
+		reg = <0x30>;
+		status = "okay";
+		clock-rate-control = <0x80>;
+		port {
+			ov2640_ep_out: endpoint {
+				remote-endpoint-label = "dvp_ep_in";
+			};
+		};
+	};
+};
+
+&dma {
+	status = "okay";
+};
+
+&lcd_cam {
+	status = "okay";
+	cam-clk = <10000000>;
+	pinctrl-0 = <&lcd_cam_default>;
+	pinctrl-names = "default";
+	source = <&ov2640>;
```

Comment:
The source device can be retrieved from the `remote-endpoint-label` (you can see https://github.com/zephyrproject-rtos/zephyr/pull/76475/commits/aea820dd2b17b9481bb7694100cc2587d1113a93) so no need to do direct reference via phandle here. This is just a recommendation, perhaps for next time.